ripuc.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+
+class RipucSpider(scrapy.Spider):
+    name = 'ripuc'
+    allowed_domains = ['ripuc.org']
+    start_urls = ['http://www.ripuc.org/eventsactions/docket.html']
+
+    def parse(self, response):
+        table = response.xpath('//td[@class="normal"]/table')
+        number_of_rows = int(float(table.xpath('count(tr)').extract_first()))
+
+        for i in range(2, number_of_rows+1):
+            if table.xpath('tr[{}]/td[@rowspan="2"][1]/a/text()'.format(i)).extract():
+                docket = table.xpath('tr[{}]/td[@rowspan="2"][1]/a/text()'.format(i)).extract()
+
+                if table.xpath('tr[{}]/td[2]/p/text()'.format(i)).extract():
```

Comment:
if we have 2 possible xpaths for a field. Can we do it this way:
`filer = xpath1 or xpath2`
instead of using if-else